plugins/scanners/scancode/src/main/kotlin/ScanCodeResultModelMapper.kt

```diff
@@ -67,8 +67,10 @@ fun ScanCodeResult.toScanSummary(preferFileLicense: Boolean = false): ScanSummar
     val header = headers.single()
     val inputPath = File(header.options.input.first())
 
-    val outputFormatVersion = header.outputFormatVersion?.let { Semver(it) }
-    if (outputFormatVersion != null && outputFormatVersion.major > MAX_SUPPORTED_OUTPUT_FORMAT_MAJOR_VERSION) {
+    // Parsing already ensured that an output format version is present.
+    val outputFormatVersion = Semver(checkNotNull(header.outputFormatVersion))
```

Comment:
Should this commit make `HeaderEntry.outputFormatVersion` non-nullable, so that this `checkNotNull` / code comment and so forth becomes redundant?